devops/openshift/forms-deploy.yml

```diff
@@ -13,6 +13,7 @@ objects:
       FORMIO_ROOT_PASSWORD: "${FORMIO_ROOT_PASSWORD}"
       MONGODB_URI: "${MONGODB_URI}"
       NODE_CONFIG: '{"mongo": "${MONGODB_URI}"}'
+      FORMS_URL: 'https://${FORMS_URL}'
```

Comment:
Can the `https` be part of `FORMS_URL`?

Reply:
i feel it cannot be done, else when forms is created the route needs just the domain. Else we need to remove the https, on all the place we try to create the forms.